e2e/pages/app/toolbar-search/toolbar-search.testpage.component.ts

```diff
@@ -30,4 +31,53 @@ export class ToolbarSearchTestPageComponent {
         // Execute search here
         this.searchedFor = this.form.controls.search.value;
     }
+
+    @ViewChild('searchField')
+    searchField: ElementRef;
+
+    toolbarExpanded: boolean;
+    toolbarSearchText: string;
+    toolbarSearchedFor: string = '';
+
+    mastheadSearchExpanded: boolean;
+    mastheadSearchText: string;
+    mastheadSearchedFor: string = '';
+    values: ReadonlyArray<string> = [];
+    searchDropdownOpen: boolean;
+
+    input: string = '';
+
+    loadOptionsFn = this.loadOptions.bind(this);
```

Comment:
Can we get rid of the load function and just use an array of a few static values